src/cacheServerClient/cacheServerClient.ts

```diff
@@ -33,9 +36,13 @@ export class CacheServerClient implements ICacheServerClient {
     private refresh_token: string | undefined;
     private readonly signer: Signer;
 
-    constructor({ url, cacheServerSupportsAuth = true }: CacheServerClientOptions, signer: Signer) {
+    constructor(
+        { url, cacheServerSupportsAuth = true, apiVersion = defaultCacheServerApiVersion }: CacheServerClientOptions,
+        signer: Signer,
+    ) {
+        console.log(`CacheServerClient.constructor url: ${url}, apiVersion: ${apiVersion}`);
         this.httpClient = axios.create({
-            baseURL: url,
+            baseURL: url.endsWith("/") ? `${url}${apiVersion}` : `${url}/${apiVersion}`,
```

Comment:
I'm just curious what you see as the advantage of having a separate `apiVersion` param over having folks add the version to the `url`?

Reply:
Separation of version from base url would give more flexibility like changing type of versioning, but it is very unlikely be needed. So probably keeping them together is simpler

Reply:
Ok, so is this PR not needed then as we can simply change the `url`?

Reply:
Whoops, nevermind, I see you've changed this